<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>1041</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Item Name</string>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Revenue</string>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>650</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Revenue Generated:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
